hudi-cli/src/test/java/org/apache/hudi/cli/integ/ITTestRepairsCommand.java

```diff
@@ -102,9 +104,12 @@ public void init() throws Exception {
     cowTable.withInserts(HoodieTestDataGenerator.DEFAULT_SECOND_PARTITION_PATH, "4", hoodieRecords1)
             .withInserts(HoodieTestDataGenerator.DEFAULT_THIRD_PARTITION_PATH, "6", hoodieRecords1);
 
+    cowTable.withInserts(HoodieTestDataGenerator.NO_PARTITION_PATH, "9", hoodieRecords1);
```

Comment:
This does not properly generate the non-partitioned table for testing.  You need to create a separate table path and `HoodieSparkWriteableTestTable` instance to generate separate non-partitioned tables for COW and MOR.  If you're not sure about how to implement this, I can help you with tests, so we can land it soon for the 0.12.1 release.
```
String cowNonPartitionedTablePath = Paths.get(basePath, HoodieTableType.COPY_ON_WRITE.name()).toString();
HoodieSparkWriteableTestTable cowNonPartitionedTable = ...
```

Reply:
Thank you for review! I pull your commit.

Reply:
@brightwon No problem.  I only rebased your PR on the latest master and haven't changed the test yet.  Let me know you can fix the test.

Reply:
better to create a separate table as @yihua suggested inside the test itself. Also, it should be sufficient to test the functionality for COW table only (saves test time).